include/seqan3/range/view/drop.hpp

```diff
@@ -0,0 +1,204 @@
+            {
+                std::move(b),
+                std::ranges::end(urange)
+
+            };
+        }
+    }
+
+    /*!\brief       Overload for contiguous, sized ranges.
+     * \returns     A std::span over the input.
+     */
+    template <std::ranges::ViewableRange urng_t>
+        requires std::ranges::ContiguousRange<urng_t> && std::ranges::SizedRange<urng_t>
+    static auto impl(urng_t && urange, size_t drop_size)
+    {
+        drop_size = std::min(drop_size, std::ranges::size(urange));
+        return std::span{std::ranges::data(urange) + drop_size, std::ranges::size(urange) - drop_size};
+    }
+
+    /*!\brief       Overload for std::basic_string_view.
+     * \returns     A std::span over the input.
+     */
+    template <std::ranges::ViewableRange urng_t>
+        requires std::ranges::ContiguousRange<urng_t> && std::ranges::SizedRange<urng_t> &&
+                 is_type_specialisation_of_v<std::remove_reference_t<urng_t>, std::basic_string_view>
+    static auto impl(urng_t && urange, size_t drop_size)
+    {
+        drop_size = std::min(drop_size, std::ranges::size(urange));
+        return urange.substr(drop_size);
+    }
+
+    /*!\brief       Overload for std::basic_string.
+     * \returns     A std::span over the input.
+     */
+    template <typename char_t, typename traits_t, typename allocator_t>
+    static std::basic_string_view<char_t, traits_t> impl(std::basic_string<char_t, traits_t, allocator_t> & urange,
+                                                         size_t drop_size)
+    {
+        drop_size = std::min(drop_size, std::ranges::size(urange));
+        return {std::ranges::data(urange) + drop_size, std::ranges::size(urange) - drop_size};
+    }
+
+    //!\overload
+    template <typename char_t, typename traits_t, typename allocator_t>
+    static std::basic_string_view<char_t, traits_t> impl(std::basic_string<char_t, traits_t, allocator_t> const & urange,
+                                                         size_t drop_size)
+    {
+        drop_size = std::min(drop_size, std::ranges::size(urange));
+        return {std::ranges::data(urange) + drop_size, std::ranges::size(urange) - drop_size};
+    }
+
+    //!\overload
+    template <typename char_t, typename traits_t, typename allocator_t>
+    static std::basic_string_view<char_t, traits_t> impl(std::basic_string<char_t, traits_t, allocator_t> const && urange,
+                                                         size_t drop_size) = delete;
+};
+
+} // namespace seqan3::detail
+
+// ============================================================================
+//  view::drop (adaptor instance definition)
+// ============================================================================
+
+namespace seqan3::view
+{
+
+/*!\name General purpose views
+ * \{
+ */
+
+/*!\brief               A view adaptor that returns all elements after n from the underlying range (or an empty range
```

Comment:
```suggestion
/*!\brief               A view adaptor that returns all elements after drop_size from the underlying range (or an empty range
```
?

Reply:
Hm, I try not to use parameter names in the brief because you see the brief without the rest often. `n` is kind of universally known so I like it for these kind of things...